src/org/zaproxy/zap/extension/codedx/CodeDxExtension.java

```diff
@@ -62,7 +63,7 @@ public boolean canUnload() {
     @Override
     public void hook(ExtensionHook extensionHook) {
         super.hook(extensionHook);
-
+        API.getInstance().registerApiImplementor(new CodeDxAPI(this));

```

Comment:
This needs to be removed when the add-on is uninstalled, if targeting ZAP 2.6.0 is not a problem it could call `extensionHook.addApiImplementor` instead (which allows core to automatically remove it), otherwise you need to override `unload` and call `removeApiImplementor`.